src/main/resources/ValidationMessages_fa_IR.properties

```diff
@@ -0,0 +1,3 @@
+size.category_characters_msg=\u0646\u0627\u0645 \u0637\u0628\u0642\u0647 \u0628\u0646\u062f\u06cc \u0628\u0627\u06cc\u062f \u062d\u062f\u0627\u0642\u0644 \u062f\u0648 \u062d\u0631\u0641 \u062f\u0627\u0634\u062a\u0647 \u0628\u0627\u0634\u062f
+size.product_characters_msg=\u0646\u0627\u0645 \u0645\u062d\u0635\u0648\u0644 \u0628\u0627\u06cc\u062f \u062d\u062f\u0627\u0642\u0644 \u062f\u0648 \u062d\u0631\u0641 \u062f\u0627\u0634\u062a\u0647 \u0628\u0627\u0634\u062f
+min.negative_amount_msg=\u0646\u0645\u06cc \u062a\u0648\u0627\u0646 \u0645\u0628\u0644\u063a \u0645\u0646\u0641\u06cc \u062f\u0631 \u0633\u0647\u0627\u0645 \u062f\u0627\u0634\u062a
```

Comment:
Validation messages are still in English

Reply:
Updated.